server/controllers/request.js

```diff
@@ -0,0 +1,38 @@
+const Request = require("../models/Request");
+const express = require("express");
+const router = express.Router();
+
+// @desc  GET : list of requests for logged in user
+// @route GET
+
+const getRequests = async (req, res) => {
+  const requests = await Request.find({ userId: req.user.id });

```

Comment:
```suggestion
  const requests = await Request.find({ $or: [{ ownerId: req.user.id }, { sitterId: req.user.id }] })

```
This query will only search for service requests which have been requested by the user, instead you can change the query as above :)